platform/lib/platform_web/live/settings_live/mfa_live.ex

```diff
@@ -58,6 +59,20 @@ defmodule PlatformWeb.SettingsLive.MFALive do
     end
   end
 
+  def handle_event("access_otp", %{"access_otp" => query}, socket) do
+    changeset = Accounts.confirm_user_mfa(socket.assigns.current_user, query)
+
+    if changeset.valid? do
+      {:noreply, socket |> assign(:otp_passed, true)}
```

Comment:
Currently you can use a backup code to pass this check but the backup code is not immediately marked as invalid in the interface.

Reply:
Fixed by applying changes and updating `current_user`